dev/tasks/mkcert.py

```diff
@@ -31,8 +33,13 @@ def up(self, args: Optional[Any], extra_args: Optional[Any]) -> None:
         key = args['key']
         crt = args['crt']
         joined_names = ' '.join(args['names'])
+        current_repo = os.path.basename(os.path.normpath(environment.env['WORKON_HOME']))
 
-        if os.path.exists(key) and os.path.exists(crt):
+        if (
+            os.path.exists(key) and
+            os.path.exists(crt) and
+            not HashCacheHelper.changed(current_repo + key, joined_names)
```

Comment:
You can use `environment.name` here instead of `current_repo`, we might also want to pick up changes to `crt`, maybe include it as well?

Reply:
`environment.name` can collide, but they should be unique enough.

Reply:
Ahh that's where it is :) I printed `environment.env` and couldn't find it lol. Yup, will add that too.